File: ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecurePaymentAuthRequestCallback.java
package com.braintreepayments.api;

import android.content.Context;

/**
 * Used to receive the result of {@link ThreeDSecureClient#createPaymentAuthRequest(Context, ThreeDSecureRequest, ThreeDSecurePaymentAuthRequestCallback)}  
 */
public interface ThreeDSecurePaymentAuthRequestCallback {

    void onThreeDSecurePaymentAuthRequest(ThreeDSecurePaymentAuthRequest paymentAuthRequest);
}

File: ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/ThreeDSecurePaymentAuthRequestCallback.kt
package com.braintreepayments.api.threedsecure

/**
 * Used to receive the result of [ThreeDSecureClient.createPaymentAuthRequest]
 */
fun interface ThreeDSecurePaymentAuthRequestCallback {

    fun onThreeDSecurePaymentAuthRequest(paymentAuthRequest: ThreeDSecurePaymentAuthRequest)
}
